pkg/podexecutor/staticpod.go

```diff
@@ -86,20 +86,30 @@ type ControlPlaneMounts struct {
 	CloudControllerManager []string
 }
 
+type ControlPlaneProbeConfs struct {
+	KubeAPIServer          staticpod.ProbeConfs
+	KubeScheduler          staticpod.ProbeConfs
+	KubeControllerManager  staticpod.ProbeConfs
+	KubeProxy              staticpod.ProbeConfs
+	Etcd                   staticpod.ProbeConfs
+	CloudControllerManager staticpod.ProbeConfs
+}
+
 type StaticPodConfig struct {
-	ManifestsDir          string
-	ImagesDir             string
-	Resolver              *images.Resolver
-	CloudProvider         *CloudProviderConfig
-	DataDir               string
-	AuditPolicyFile       string
-	KubeletPath           string
-	ControlPlaneResources ControlPlaneResources
-	ControlPlaneMounts    ControlPlaneMounts
-	ControlPlaneEnv       ControlPlaneEnv
-	CISMode               bool
-	DisableETCD           bool
-	IsServer              bool
+	ManifestsDir    string
+	ImagesDir       string
+	Resolver        *images.Resolver
+	CloudProvider   *CloudProviderConfig
+	DataDir         string
+	AuditPolicyFile string
+	KubeletPath     string
+	CISMode         bool
+	DisableETCD     bool
+	IsServer        bool
+	ControlPlaneResources
```

Comment:
Generally, convention is to have the embedded structs at the top of the struct. Less important, it's best to keep the bools at the bottom.

Reply:
done, thanks!